build/vue-loader.conf.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       ? config.build.productionSourceMap
       : config.dev.cssSourceMap,
     extract: isProduction
-  }), { js: 'vue-ts-loader' }),
+  }), { js: ['../../loaders/standard-loader', 'vue-ts-loader'] }),
```

Comment:
this relative path calc from actual vue filepath.
That's why I moved App.vue file.